pages/k8s/operations.md

```diff
@@ -0,0 +1,278 @@
+  results:
+    address: microbot.52.87.186.136.xip.io
+  status: completed
+  timing:
+    completed: 2019-03-22 15:00:39 +0000 UTC
+    enqueued: 2019-03-22 15:00:34 +0000 UTC
+    started: 2019-03-22 15:00:37 +0000 UTC
+  unit: kubernetes-worker/0
+```
+
+At this point, you can inspect the cluster to observe the workload coming
+online.
+
+### List the pods
+
+```bash
+kubectl get pods
+```
+
+```
+    NAME                        READY   STATUS    RESTARTS   AGE
+    microbot-5b9864df4d-q7b94   1/1     Running   0          2m31s
+    microbot-5b9864df4d-rx9b2   1/1     Running   0          2m31s
+    microbot-5b9864df4d-x7ppr   1/1     Running   0          2m31s
+```
+
+
+
+### List the services and endpoints
+
+```bash
+kubectl get services,endpoints
+```
+```
+NAME                 TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)   AGE
+service/kubernetes   ClusterIP   10.152.183.1    <none>        443/TCP   112m
+service/microbot     ClusterIP   10.152.183.62   <none>        80/TCP    3m50s
+
+NAME                   ENDPOINTS                                   AGE
+endpoints/kubernetes   10.95.195.54:6443                           112m
+endpoints/microbot     10.1.77.12:80,10.1.77.13:80,10.1.77.14:80   3m50s
+
+```
+
+### List the ingress resources
+
+```bash
+kubectl get ingress
+```
+```
+NAME               HOSTS                           ADDRESS   PORTS   AGE
+microbot-ingress   microbot.52.87.186.136.xip.io             80      5m36s
+```
+
+When all the pods are listed as Running, you are ready to visit the address listed in the
+HOSTS column of the ingress listing.
+
+<div class="p-notification--positive">
+  <p markdown="1" class="p-notification__response">
+    <span class="p-notification__status">Note:</span>
+It is normal to see a 502/503 error during initial application deployment
+  </p>
+</div>
+
+As you refresh the page, you will be greeted with a microbot web page, serving
+from one of the microbot replica pods. Refreshing will show you another
+microbot with a different hostname as the requests are load-balanced across
+the replicas.
+
+### Clean up example
+
+There is also an action to clean up the microbot applications. When you are
+done using the microbot application you can delete them from the pods with
+one Juju action:
+
+```bash
+juju run-action kubernetes-worker/0 microbot delete=true
+```
+
+If you no longer need Internet access to your workers, remember to unexpose the
+kubernetes-worker charm:
+
+```bash
+juju unexpose kubernetes-worker
+```
+
+To learn more about
+[Kubernetes Ingress](https://kubernetes.io/docs/user-guide/ingress.html)
+and how to configure the Ingress Controller beyond defaults (such as TLS and
+websocket support) view the
+[nginx-ingress-controller](https://github.com/kubernetes/contrib/tree/master/ingress/controllers/nginx)
+project on github.
+
+
+
+<a id='next'> </a>
+
+## Next steps
+
+Now your **CDK** cluster is up and running, here are some suggestions for additional
```

Comment:
Again, should this be "Now that your..." instead? Maybe this is a British/American English thing, I dunno.

Reply:
either would be okay in GB (i.e. real) English, but if it sounds odd to your ears, I will add a few 'that's